eng/pipelines/azure-pipelines.yml

```diff
@@ -98,7 +98,7 @@ stages:
       jobs:
 
       - job: windows
-        timeoutInMinutes: 30
+        timeoutInMinutes: 45
```

Comment:
😞

Reply:
FWIW, 30 mins was already pretty low and super close to the time it actually takes to build the repo in AzDO, so we were going to have to do this sooner rather than later.

Reply:
![image](https://github.com/dotnet/aspire/assets/8291187/366bbf8c-92ac-413e-aee2-922ad5b551ed)

that's from a recent CI run.